cl/search/api_serializers.py

```diff
@@ -151,7 +156,7 @@ class OpinionSerializer(DynamicFieldsMixin, HyperlinkedModelSerializerWithId):
 
     class Meta:
         model = Opinion
-        fields = "__all__"
+        exclude = ("local_path",)
```

Comment:
Sorry if this wasn't clear, but local_path *is* important because it shows people where to find the PDFs. Here's an example:

        {
            "resource_uri": "https://www.courtlistener.com/api/rest/v3/opinions/4468530/",
            "id": 4468530,
            "absolute_url": "/opinion/4691277/liliana-mercado-ventura-v-william-barr-u/",
            "cluster": "https://www.courtlistener.com/api/rest/v3/clusters/4691277/",
            "author": null,
            "joined_by": [],
            "author_str": "",
            "per_curiam": false,
            "joined_by_str": "",
            "date_created": "2019-12-31T19:00:19.651375Z",
            "date_modified": "2019-12-31T19:00:19.759216Z",
            "type": "010combined",
            "sha1": "ed031f3681ad0c0f7316c628765156ebcecf5ee5",
            "page_count": 3,
            "download_url": "http://www.ca5.uscourts.gov/opinions/unpub/19/19-60022.0.pdf",
            "local_path": "pdf/2019/12/31/liliana_mercado-ventura_v._william_barr_u..pdf",
            "plain_text": "     Case: 19-60022       Document: 00515253354         Page: 1     Date Filed: 12/31/2019\n\n\n\n\n           IN THE UNITED STATES COURT OF APPEALS\n                    FOR THE FIFTH CIRCUIT\n                                                                          United States Court of Appeals\n                                                                                   Fifth Circuit\n\n\n                                     No. 19-60022\n                                                                                 FILED\n                                                                         December 31, 2019\n                                   Summary Calendar\n                                                                            Lyle W. Cayce\n                                                                                 Clerk\nLILIANA CAROLINA MERCADO-VENTURA; LEANDRO SEBASTIAN\nVANEGAS-MERCADO,\n\n                                                  Petitioners\n\nv.\n\nWILLIAM P. BARR, U. S. ATTORNEY GENERAL,\n\n                                                  Respondent\n\n\n                       Petitions for Review of an Order of the\n                           Board of Immigration Appeals\n                               BIA Nos. A206 798 110\n                                         A206 798 111\n\n\nBefore BARKSDALE, ELROD, and DUNCAN, Circuit Judges.\nPER CURIAM: *\n       Liliana Carolina Mercado-Ventura and her minor son, natives and\ncitizens of El Salvador, petition for review of the Board of Immigration Appeals’\n(BIA) denying their challenge to an Immigration Judge’s (IJ) denial of their\napplications for asylum and withholding of removal. Petitioners contend: their\ntestimony before the IJ sufficiently raised a familial-based particular social\n\n\n       * Pursuant to 5th Cir. R. 47.5, the court has determined that this opinion should not\nbe published and is not precedent except under the limited circumstances set forth in 5th Cir.\nR. 47.5.4.\n\f    Case: 19-60022     Document: 00515253354      Page: 2   Date Filed: 12/31/2019\n\n\n                                  No. 19-60022\n\ngroup (in addition to the particular social group the IJ considered); and the\nBIA abused its discretion, and denied them a full and fair hearing, by failing\nto remand the case for the IJ to further consider this additional particular\nsocial group.\n      To the extent the BIA relied upon the IJ’s decision, we may review the\ndecisions of both the BIA and the IJ. See Efe v. Ashcroft, 293 F.3d 899, 903\n(5th Cir. 2002) (citation omitted). “We review factual findings of the BIA and\nIJ for substantial evidence, and questions of law de novo . . . .” Zhu v. Gonzales,\n493 F.3d 588, 594 (5th Cir. 2007) (citation omitted). The determination an\nalien is ineligible for asylum or withholding of removal is a factual finding.\nTamara-Gomez v. Gonzales, 447 F.3d 343, 347 (5th Cir. 2006) (citation\nomitted).   Whether a proposed particular social group is cognizable for\npurposes of asylum and withholding of removal is a question of law. See\nHernandez-De La Cruz v. Lynch, 819 F.3d 784, 786 (5th Cir. 2016) (citation\nomitted).\n      To qualify for asylum, an alien must show that (1) she was persecuted or\nhas a well-founded fear of persecution, (2) “by the government or forces that a\ngovernment is unable or unwilling to control”, (3) on account of a protected\nground, including membership in a particular social group. Tesfamichael v.\nGonzales, 469 F.3d 109, 113 (5th Cir. 2006) (citations omitted). “The standard\nfor obtaining withholding of removal is even higher than the standard for\nasylum, requiring a showing that it is more likely than not that the alien’s life\nor freedom would be threatened by persecution on one of those [protected]\ngrounds.”   Orellana-Monson v. Holder, 685 F.3d 511, 518 (5th Cir. 2012)\n(citation omitted).\n      An alien has the burden of proof and the duty of delineating the contours\nof the proposed particular social group before the IJ. See Matter of W-Y-C- and\n\n\n\n                                        2\n\f    Case: 19-60022     Document: 00515253354     Page: 3   Date Filed: 12/31/2019\n\n\n                                  No. 19-60022\n\nH-O-B-, 27 I. & N. Dec. 189, 191 (BIA 2018) (citations omitted). In this case,\ncounsel clearly delineated for the IJ the proposed social group as “Salvadoran\nwomen . . . who fear gangs and violence in their home country” and affirmed\nthat particular social group when questioned by the IJ. Counsel’s statement\nof the particular social group was clear and unequivocal; the IJ, therefore, was\nnot obligated to seek further clarification. See id.\n      A different or narrower proposed social group presented on appeal to the\nBIA is not sufficiently raised before the IJ and need not be considered by the\nBIA. Id. at 191–92 (citations omitted). The BIA, therefore, did not err in\nfinding counsel failed to raise the alternative group before the IJ and declining\nto consider it.\n      Additionally, we agree with the Government that petitioners abandoned\nany challenge to the BIA’s denial of relief based on the particular social group\npresented to the IJ by failing to adequately brief the issue. See Soadjede v.\nAshcroft, 324 F.3d 830, 833 (5th Cir. 2003) (per curiam) (citation omitted).\nPetitioners, therefore, have not established eligibility for asylum and,\ntherefore, cannot do so for the higher withholding-of-removal standard.\n      DENIED.\n\n\n\n\n                                        3\n\f",
            "html": "",
            "html_lawbox": "",
            "html_columbia": "",
            "xml_harvard": "",
            "html_with_citations": "",
            "extracted_by_ocr": false,
            "opinions_cited": []
        },

Without that, it's unclear how to find the PDFs. A better field would be to show the actual URL for the PDFs, but since we lack that this is what folks use.